packages/darklang/stdlib/string.dark

```diff
@@ -176,6 +309,23 @@ module Darklang =
         == suffix
 
 
+      /// Returns {{Just index}} of the first occurrence of <param searchFor> in <param str>, or returns {{Nothing}} if <param searchFor> does not occur.
+      let indexOf
+        (str: String)
+        (searchFor: String)
+        : PACKAGE.Darklang.Stdlib.Option.Option<Int> =
+        Builtin.String.indexOf str searchFor
+
+
+      /// Returns the <param string> repeated for a specified <param count> times.
+      /// If <param count> is less than or equal to 0, returns an empty string.
+      let repeat (string: String) (count: Int) : String =
+        if count <= 0 then
+          ""
+        else
+          string ++ PACKAGE.Darklang.Stdlib.String.repeat string (count - 1)
```

Comment:
This is an O(N^2) function. You can make it O(N) by using a list and calling String.join.